packages/replay-next/src/suspense/SearchCache.ts

```diff
@@ -69,18 +71,34 @@ export function assertSourceSearchResultMatch(
 }
 
 export const searchCache = createStreamingCache<
-  [replayClient: ReplayClientInterface, query: string, includeNodeModules: boolean, limit?: number],
+  [
+    replayClient: ReplayClientInterface,
+    query: string,
+    defaultFilter: boolean,
+    includedFiles: string,
+    excludedFiles: string,
+    openSourceIds: string[] | null,
```

Comment:
nit: When I first read this I was confused about what "open source" meant (like node_modules)

 I wonder if a name like "active sources" would be less confusing/ambiguous than "open sources"?

Reply:
I agree, this is something that confused me at first when navigating the codebase as well. I kept it `openSourceIds` as that is what `SourceContext` is also naming it as. Maybe it makes sense changing there as well?

Reply:
> Maybe it makes sense changing there as well?

👍🏼 Yeah I dig that.